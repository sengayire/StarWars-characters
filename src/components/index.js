import Login from './Login';
import Profile from './Profile';
import UserList from './UserList';

export{Login, Profile, UserList};
