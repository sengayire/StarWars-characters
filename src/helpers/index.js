import apiAction from './apiAction';
import axiosHelper from './axiosHelper';
import userList from './userList';
import * as urlHelper from './urlHelper';

export { apiAction, axiosHelper, urlHelper, userList };
