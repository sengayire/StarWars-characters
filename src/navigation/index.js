import React from 'react'; 
import { NavigationContainer } from '@react-navigation/native';
import { createStackNavigator } from '@react-navigation/stack';
import {Login} from '../screens';

const Stack = createStackNavigator();

const Navigation = () => {
  return (
    <NavigationContainer>
      <Stack.Navigator>
        <Stack.Screen name="Welcome" component={Login} />
      </Stack.Navigator>
    </NavigationContainer>
  );
}

export default Navigation;