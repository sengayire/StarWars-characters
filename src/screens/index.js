import  Login from './Login';
import Profile from './Profile';
import UserList from './UserList';
import UserInfo from './UserInfo';

export {Login, Profile, UserList, UserInfo};
