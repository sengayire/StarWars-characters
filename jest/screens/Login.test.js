import React from 'react';
import {shallow} from 'enzyme';
import {Login} from '../../src/screens';

describe('Login', () => {
    it('should match to snapshot - Secondary', () => {
        const component = shallow(<Login />)
        expect(component).toMatchSnapshot()
    });
});
