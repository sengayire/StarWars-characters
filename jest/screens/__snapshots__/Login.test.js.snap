// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should match to snapshot - Secondary 1`] = `
<View>
  <Text>
    Welcome to star wars
  </Text>
</View>
`;
